packages/plugin-ext-vscode/src/browser/plugin-vscode-commands-contribution.ts

```diff
@@ -54,5 +68,30 @@ export class PluginVscodeCommandsContribution implements CommandContribution {
                 this.contextKeyService.createKey(String(contextKey), contextValue);
             }
         });
+
+        commands.registerCommand(VscodeCommands.PREVIEW_HTML, {
+            isVisible: () => true,
+            // tslint:disable-next-line: no-any
+            execute: (resource: URI, position?: any, label?: string, options?: any) => {
+                label = label || resource.fsPath;
+                const view = new WebviewWidget(label, {allowScripts: true}, {});
+                this.resorces(new TheiaURI(resource)).then(res => {
+                    res.readContents().then(str => {
+                        const html = '<!DOCTYPE html><html><head></head>' + str + '</html>';
```

Comment:
could you replace by 
```suggestion
                        const html = `<!DOCTYPE html><html><head></head>${str}</html>`;
```

Reply:
yep, i tried it but it won't work for me

Reply:
hello Vitalii, why it won't work ?

Reply:
I got not resolved template:
 ```
private getHtml(body: String) {
        return '<!DOCTYPE html><html><head></head>${body}</html>';
    }
```
return
![screenshot-localhost-3000-2019 02 19-17-09-41](https://user-images.githubusercontent.com/1636592/53025216-2d6f7380-3469-11e9-8626-024a647bfa48.png)

Reply:
it's backtick : `
`
` not a single quote

Reply:
:( ups. Got it